ocs_ci/deployment/hub_spoke.py

```
@@ -4352,6 +4357,72 @@ def destroy_aws_hcp_cluster(self, timeout=1200):
+            )
+            if secret_obj.is_exist():
+                secret_data = secret_obj.get()
+                encoded = secret_data.get("data", {})
+                bucket = base64.b64decode(encoded.get("bucket", "")).decode()
+                region = base64.b64decode(encoded.get("region", "")).decode()
+                if bucket:
+                    self.oidc_bucket_name = bucket
+                    self.oidc_bucket_region = region or self.aws_region
+                    logger.info(
+                        f"OIDC bucket from K8s secret '{secret_name}': "
+                        f"{self.oidc_bucket_name} in region {self.oidc_bucket_region}"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)